l10n-pocket/en/brands.ftl

```diff
@@ -57,3 +57,5 @@
 ## Google Products
 
 -brand-name-google-play = Google Play
+-brand-name-google-analytics = Google Analytics
```

Comment:
Question / Suggestion (non-blocking): are we likely to use the string `Google Analytics` elsewhere in Pocket pages?

There's no harm in adding this, however `brands.ftl` is primarily for commonly used brand names (e.g. ones that appear repeatedly throughout our pages). There's no hard rule that says every brand or product name we mention needs to be in here. Using something like `{ -brand-name-google } Analytics` in an`.ftl` string is probably OK.

I'll leave this up to you if you want to add, just calling out

Reply:
Ah, I was thinking that what was in `brands.ftl` were strings that shouldnt be translated (since they would be the same in each language?) Thanks for clarifying.